client/components/product-card/action.jsx

```diff
@@ -22,6 +28,14 @@ ProductCardAction.propTypes = {
 	intro: PropTypes.oneOfType( [ PropTypes.string, PropTypes.node, PropTypes.element ] ),
 	label: PropTypes.string.isRequired,
 	onClick: PropTypes.func,
+	href: PropTypes.string,
+	isPrimary: PropTypes.boolean,
+};
+
+ProductCardAction.defaultProps = {
+	href: null,
```

Comment:
Could you please document these new `href` and `primary` props in the README of the `ProductCard` component? The rest are already documented.